etc/cmake/options.cmake

```diff
@@ -229,9 +229,14 @@ if(OT_LINK_RAW)
     target_compile_definitions(ot-config INTERFACE "OPENTHREAD_CONFIG_LINK_RAW_ENABLE=1")
 endif()
 
-option(OT_LINK_METRICS "enable link metrics")
-if (OT_LINK_METRICS)
-    target_compile_definitions(ot-config INTERFACE "OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE=1")
+option(OT_LINK_METRICS_INITIATOR "enable link metrics initiator")
```

Comment:
Can you please take care of this change in the `zephyr/CMakeLists.txt` file?

Reply:
Good point. Added necessary changes.